bitnami/nginx/README.md

```diff
@@ -42,41 +42,43 @@ The command removes all the Kubernetes components associated with the chart and
+| `ingress.enabled`                | Enable ingress controller resource               | `false`                                                                                                                                      |
+| `ingress.certManager`            | Add annotations for cert-manager                 | `false`                                                                                                                                      |
+| `ingress.annotations`            | Ingress annotations                              | `[]`                                                                                                                                         |
+| `ingress.hosts[0].name`          | Hostname to your NGINX installation              | `nginx.local`                                                                                                                                |
+| `ingress.hosts[0].path`          | Path within the url structure                    | `/`                                                                                                                                          |
+| `ingress.tls[0].hosts[0]`        | TLS hosts                                        | `nginx.local`                                                                                                                                |
+| `ingress.tls[0].secretName`      | TLS Secret (certificates)                        | `nginx.local-tls`                                                                                                                            |
+| `ingress.secrets[0].name`        | TLS Secret Name                                  | `nil`                                                                                                                                        |
+| `ingress.secrets[0].certificate` | TLS Secret Certificate                           | `nil`                                                                                                                                        |
+| `ingress.secrets[0].key`         | TLS Secret Key                                   | `nil`                                                                                                                                        |
+| `livenessProbe`                  | Deployment Liveness Probe                        | `httpGet:`<br>&nbsp;&nbsp;`path: /`<br>&nbsp;&nbsp;`port: http`<br>`initialDelaySeconds: 30`<br>`timeoutSeconds: 5`<br>`failureThreshold: 6` |
+| `readinessProbe`                 | Deployment Readiness Probe                       | `httpGet:`<br>&nbsp;&nbsp;`path: /`<br>&nbsp;&nbsp;`port: http`<br>`initialDelaySeconds: 5`<br>`timeoutSeconds: 3`<br>`periodSeconds: 5`     |
```

Comment:
This format is a little illegible, I'd refer to the **values.yaml**. E.g.:

```
| `readinessProbe`       | Deployment Readiness Probe        | See `values.yaml`                        |
```

Reply:
Sounds good. I'll get this done.